packages/ui-new/src/composites/wui-link/styles.ts

```diff
@@ -1,19 +1,67 @@
-import { css } from 'lit'
+import { css } from '../../utils/ThemeHelperUtil.js'
 
 export default css`
   button {
-    padding: var(--wui-spacing-4xs) var(--wui-spacing-xxs);
-    border-radius: var(--wui-border-radius-3xs);
-    background-color: transparent;
-    color: var(--wui-color-accent-100);
+    border: none;
+    background: transparent;
+    height: 20px;
+    padding: ${({ spacing }) => spacing[2]};
+    column-gap: ${({ spacing }) => spacing[1]};
+    border-radius: ${({ borderRadius }) => borderRadius[1]};
+    padding: 0 ${({ spacing }) => spacing[1]};
+    border-radius: ${({ spacing }) => spacing[1]};
   }
 
+  /* -- Variants --------------------------------------------------------- */
+  button[data-variant='accent'] {
+    color: ${({ tokens }) => tokens.core.textAccentPrimary};
+  }
+
+  button[data-variant='secondary'] {
+    color: ${({ tokens }) => tokens.theme.textSecondary};
+  }
+
+  /* -- Focus states --------------------------------------------------- */
+  button[data-variant='accent']:focus-visible:enabled {
+    background-color: ${({ tokens }) => tokens.core.foregroundAccent010};
+    box-shadow: 0px 0px 0px 4px rgba(9, 136, 240, 0.2);
```

Comment:
Let's move the all button's disabled and focus states to root instead of defining in every component separately